### PR TITLE
Add the missing HTML attributes

### LINKED ISSUE #539: Documentations issues
Missing example:
* [x] https://camptocamp.github.io/ngeo/master/apidoc/gmf.themeselectorDirective.html
* [x] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.popupDirective.html

Missing some @htmlAttribute:
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/gmf.layertreeDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/gmf.mapDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/gmf.mobileBackgroundLayerSelectorDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.btngroupDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.controlDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.desktopGeolocationDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.filereaderDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.layertreeDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.mapDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.mobileGeolocationDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.profileDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.recenterDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.scaleselectorDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.searchDirective.html
* [ ] https://camptocamp.github.io/ngeo/master/apidoc/ngeo.sortableDirective.html

### PR DESCRIPTION
**Pull request by sbrunner**
I needs some help to fill all the `???`...
Thanks in advance

Partially fix #539

### CODE REVIEW

File: src/directives/filereader.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ goog.require('ngeo');
  * <input type="file" ngeo-filereader="ctrl.fileContent"
  *        ngeo-filereader-supported="ctrl.supported"/>
  *
+ * @htmlAttribute {???} ngeo-filereader ???
```
Comment (gberaudo) [inline src/directives/filereader.js]: string: the content of the file read.
Is this directive useful?

File: src/directives/filereader.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ goog.require('ngeo');
...
+ * @htmlAttribute {???} ngeo-filereader-supported ???
```
Comment (gberaudo) [inline src/directives/filereader.js]: boolean: if the FileReader API is supported?

File: src/directives/profile.js [hunk 3]
```diff
@@ -21,6 +21,9 @@ goog.require('ngeo.profile');
  * processed by {@link ngeox.profile.ElevationExtractor} and
  * {@link ngeox.profile.PoiExtractor}.
  *
+ * @htmlAttribute {???} ngeo-profile ???
+ * @htmlAttribute {ngeox.profile.ProfileOptions} ngeo-profile-options The options
+ * @htmlAttribute {???} ngeo-profile-pois ???
```
Comment (gberaudo) [inline src/directives/profile.js]: Missing ngeoProfileHighlight.

File: contribs/gmf/src/directives/layertree.js [hunk 4]
```diff
@@ -59,6 +59,8 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
  * metadata URLs in a new window. By default, and in the default template,
  * links will be opened in a popup.
  *
+ * @htmlAttribute {???} gmf-layertree-source ???
```
Comment (ger-benjamin) [inline contribs/gmf/src/directives/layertree.js]: {Object} That's one "theme" (JSON)

File: src/directives/layertree.js [hunk 5]
```diff
@@ -75,6 +75,11 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
  * controller: "layertreeCtrl". You can refer to that property in a custom
  * template for example.
  *
+ * @htmlAttribute {???} ngeo-layertree The tree
```
Comment (ger-benjamin) [inline src/directives/layertree.js]: {Object} That's one "theme" (JSON)

File: src/directives/layertree.js [hunk 6]
```diff
@@ -75,6 +75,11 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
...
+ * @htmlAttribute {string} ngeo-layertree-templateurl The template url
+ * @htmlAttribute {ol.Map} ngeo-layertree-map The map
+ * @htmlAttribute {???} ngeo-layertree-nodelayer ???
+ * @htmlAttribute {???} ngeo-layertree-listners ???
```
Comment (ger-benjamin) [inline src/directives/layertree.js]: listners -> listener

{string} (expression that will be parsed, like "gmfLayertreeCtrl.listeners(treeScope, treeCtrltreeCtrl)")

File: src/directives/layertree.js [hunk 7]
```diff
@@ -75,6 +75,11 @@ ngeo.module.value('ngeoLayertreeTemplateUrl',
  * controller: "layertreeCtrl". You can refer to that property in a custom
  * template for example.
  *
+ * @htmlAttribute {???} ngeo-layertree The tree
+ * @htmlAttribute {string} ngeo-layertree-templateurl The template url
+ * @htmlAttribute {ol.Map} ngeo-layertree-map The map
+ * @htmlAttribute {???} ngeo-layertree-nodelayer ???
```
Comment (ger-benjamin) [inline src/directives/layertree.js]: {string} (expression that will be parsed, like "gmfLayertreeCtrl.getLayer(node, depth)")

File: src/directives/filereader.js [hunk 8]
```diff
@@ -14,6 +14,8 @@ goog.require('ngeo');
  * <input type="file" ngeo-filereader="ctrl.fileContent"
  *        ngeo-filereader-supported="ctrl.supported"/>
  *
+ * @htmlAttribute {string} ngeo-filereader The content of the file read
+ * @htmlAttribute {boolean} ngeo-filereader-supported Is the FileReader API is supported
```
Comment (gberaudo) [inline src/directives/filereader.js]: if

File: src/directives/mobilegeolocation.js [hunk 9]
```diff
@@ -20,6 +20,9 @@ goog.require('ol.geom.Point');
  *   ngeo-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
  * </button>
  *
+ * @htmlAttribute {???} ngeo-mobile-geolocation ???
```
Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: @adube Can you help me to fill it?

File: src/directives/profile.js [hunk 10]
```diff
@@ -21,6 +21,10 @@ goog.require('ngeo.profile');
  * processed by {@link ngeox.profile.ElevationExtractor} and
  * {@link ngeox.profile.PoiExtractor}.
  *
+ * @htmlAttribute {???} ngeo-profile ???
```
Comment (sbrunner) [inline src/directives/profile.js]: @pgiraud can you help me to fill them?

File: src/directives/btngroup.js [hunk 11]
```diff
@@ -28,6 +28,7 @@ goog.require('ngeo');
  *
  *     <div ngeo-btn-group ngeo-btn-group-active="ctrl.drawToolActive">
  *
+ * @htmlAttribute {???} ngeo-btn-group-active ???
```
Comment (sbrunner) [inline src/directives/btngroup.js]: @pgiraud Can you help me to fill it ?

File: src/directives/btngroup.js [hunk 12]
```diff
@@ -124,6 +125,7 @@ ngeo.module.controller('ngeoBtnGroupController', ngeo.BtnGroupController);
  * This example is about creating a Bootstrap button that can pressed/depressed
  * to activate/deactivate an OpenLayers 3 interaction.
  *
+ * @htmlAttribute {???} ng-model ???
```
Comment (sbrunner) [inline src/directives/btngroup.js]: @pgiraud Can you help me to fill it ?

File: src/directives/mobilegeolocation.js [hunk 13]
```diff
@@ -21,6 +21,9 @@ goog.require('ol.geom.Point');
  *        ngeo-mobile-geolocation-options="ctrl.mobileGeolocationOptions">
  *      </button>
  *
+ * @htmlAttribute {???} ngeo-mobile-geolocation ???
```
Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: @adube Can you help me to fill it?

Comment (adube) [inline src/directives/mobilegeolocation.js]: I don't understand the question.

Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: What's the type and the usage of the attribute `ngeo-mobile-geolocation`?

Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: Result example: https://github.com/camptocamp/ngeo/pull/679/files#diff-a7ce5e249befd38760e15273e744974aR64

Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: Or in the example that `ngeo-mobile-geolocation=""` should be `ngeo-mobile-geolocation`?

Comment (adube) [inline src/directives/mobilegeolocation.js]: It should be `ngeo-mobile-geolocation`.

Comment (sbrunner) [inline src/directives/mobilegeolocation.js]: OK, thanks :-)

File: src/directives/btngroup.js (hunk 12 above)
Comment (pgiraud) [inline src/directives/btngroup.js]: @htmlAttribute {*} ng-model Any property on the scope. Ideally a boolean.

File: src/directives/profile.js (hunk 10 above)
Comment (pgiraud) [inline src/directives/profile.js]: @htmlAttribute {?Object} ngeo-profile The profile data.

File: src/directives/profile.js [hunk 14]
```diff
@@ -21,6 +21,10 @@ goog.require('ngeo.profile');
...
+ * @htmlAttribute {ngeox.profile.ProfileOptions} ngeo-profile-options The options
+ * @htmlAttribute {???} ngeo-profile-pois ???
+ * @htmlAttribute {???} ngeo-profile-highlight ???
```
Comment (pgiraud) [inline src/directives/profile.js]: @htmlAttribute {*} ngeo-profile-highlight Any property on the scope which evaluated value may correspond to distance from origin.

File: src/directives/profile.js (hunk 3 above)
Comment (pgiraud) [inline src/directives/profile.js]: @htmlAttribute {?Array<Object>} ngeo-profile-pois The data for POIs.

File: src/directives/btngroup.js (hunk 11 above)
Comment (pgiraud) [inline src/directives/btngroup.js]: @htmlAttribute {*} ngeo-btn-group-active Any property of the scope. Tells whether at least one button of the group is active.

File: src/directives/btngroup.js (hunk 12 above)
Comment (sbrunner) [inline src/directives/btngroup.js]: Thanks :-)

File: src/directives/profile.js (hunk 14 above)
Comment (sbrunner) [inline src/directives/profile.js]: Thanks @pgiraud :-)

**Pull request closed without merging**